cdc/owner.go

```diff
@@ -72,6 +72,26 @@ func (o *ownership) inc() {
 	}
 }
 
+type minGCSafePointCacheEntry struct {
+	ts          model.Ts
+	lastUpdated time.Time
+}
+
+func (o *Owner) getGcMinSafePointCache(ctx context.Context) (model.Ts, error) {
+	log.Warn("Time", zap.Time("lastUpdated", o.minGCSafePointCache.lastUpdated))
+	log.Warn("After", zap.Bool("lastupdate time after 2 second", time.Now().After(o.minGCSafePointCache.lastUpdated.Add(MinGCSafePointCacheUpdateInterval))))
```

Comment:
the logs are used for debug? if not, if get the warn log what is the next move.  **zap.Bool("lastupdate time after 2 second"** , may be should not hardcode to 2, MinGCSafePointCacheUpdateInterval will not always be 2 second. btw why set MinGCSafePointCacheUpdateInterval to 2 instead of 3/4

Reply:
Yes, it is for debug. I forgot to delete it.